frontend/src/pages/Graphing/components/BarChart.tsx

```diff
@@ -107,93 +109,102 @@ export const BarChart = ({
 	)
 
 	return (
-		<ResponsiveContainer ref={chartRef}>
-			<RechartsBarChart
-				data={data}
-				barCategoryGap={1}
-				onMouseDown={onMouseDown}
-				onMouseMove={onMouseMove}
-				onMouseUp={onMouseUp}
-				onMouseLeave={onMouseLeave}
-				style={tooltipCanFreeze ? { cursor: 'pointer' } : undefined}
-			>
-				{referenceArea}
-				{children}
-				<XAxis
-					dataKey={xAxisMetric}
-					fontSize={10}
-					tick={(props: any) => (
-						<CustomXAxisTick
-							x={props.x}
-							y={props.y}
-							payload={props.payload}
-							tickFormatter={xAxisTickFormatter}
-						/>
-					)}
-					tickFormatter={xAxisTickFormatter}
-					tickLine={{ visibility: 'hidden' }}
-					axisLine={{ visibility: 'hidden' }}
-					height={12}
-					domain={['dataMin', 'dataMax']}
-					hide={showXAxis === false}
-				/>
-
-				{tooltip}
-
-				<YAxis
-					fontSize={10}
-					tickLine={{ visibility: 'hidden' }}
-					axisLine={{ visibility: 'hidden' }}
-					tick={(props: any) => (
-						<CustomYAxisTick
-							y={props.y}
-							payload={props.payload}
-							tickFormatter={yAxisTickFormatter}
-						/>
-					)}
-					tickFormatter={yAxisTickFormatter}
-					tickCount={7}
-					width={32}
-					type="number"
-					hide={showYAxis === false}
-				/>
-
-				{showGrid && (
-					<CartesianGrid
-						strokeDasharray=""
-						vertical={false}
-						stroke="var(--color-gray-200)"
+		<span onMouseOver={onMouseOver}>
```

Comment:
`onMouseEnter` does not always fire due to quickly moving over and blocking child elements, result in the tooltip not being displayed. Recharts charts and ResponsiveContainer does not support `onMouseOver`, so add a wrapper to detect when the mouse enters the chart